Source/Models/APIClient.swift

```diff
@@ -4,46 +4,35 @@ import Result
 
 public struct APIClient {
   private let requestPerformer: RequestPerformer
+  public let deserializer: Deserializer
```

Comment:
Does that actually matter for the `Client` protocol? Does `deserializer`? Or are these both implementation details of our default `APIClient`. I'd argue the latter.